### PR TITLE
modified theme.css file

### PR DESCRIPTION
**Pull request by AmJustSam**
Modified theme.css file + made navigation responsive for small media devices.

### CODE REVIEW

File: public/css/theme.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url(http://fonts.googleapis.com/css?family=Lobster|Abel|Shadows+Into+Light|Pacifico|Orbitron);
```
Comment (PeterDaveHello) [inline public/css/theme.css]: Please use https here

**Comment by PeterDaveHello** [comment]
Thanks @Klo1029, would you mind to give us screenshots before and after your modification? Thanks.

**Comment by Klo1029** [comment]
Just click above on modified theme.css

It will show the code I added in green background as highlighted code.

**Comment by redox** [comment]
> It will show the code I added in green background as highlighted code.

Sure thing, but having a screenshot is easier to understand the CSS diff :)

**Comment by Klo1029** [comment]
code i added

File: public/css/theme.css (hunk 1 above)
Comment (redox) [inline public/css/theme.css]: Also, why loading all those fonts? Especially because you're using `Abel` only :/

**Comment by Klo1029** [comment]
I've done media queries work as you can see in the screenshot !

**Comment by redox** [comment]
Oh, we were speaking about a screenshot of the resulting website, not of your code :smile:

**Comment by Klo1029** [comment]
screenshot 1
screenshot2

**Comment by Klo1029** [comment]
Here it is the screenshot of result after modification!

**Comment by redox** [comment]
Alright :) I think there are still room for improvements.

My 2 cts:
 - the additional fonts are not required
 - the green buttons are not really matching the overall design
 - the menu can probably be hidden in a "fa fa-bars" like icon and only be displayed if the user click on it

**Comment by Klo1029** [comment]
ok

**Comment by PeterDaveHello** [comment]
Agree with @redox

**Comment by Klo1029** [comment]
modified part 1

=====================================

modified part 2

**Comment by Klo1029** [comment]
Made changes that you asked for !

**Comment by PeterDaveHello** [comment]
What do you think @redox @cdnjs/team-cdnjs @cdnjs/intern ?

**Comment by redox** [comment]
Hi @Klo1029,

thank you for your help. I'm afraid we cannot merge that as-is as there are too many issues:
 * why using a new font?
 * why loading a new jquery? (the version 1.7.2 is already included)
 * for the menu, I would rather use Bootstrap's CSS rules instead of the home-made JS-based toggle
 * too many CSS rules overriding global rules `.show`, `btn....`, etc...

What do you think?

**Comment by Piicksarn** [comment]
I think the view will be simple without horizontal scroll bar.
As the picture shown below 

e7710eda-4834-11e5-95c9-43061cb2d0b0

But my thought will cause the size of font to be small.:disappointed_relieved:

**Comment by PeterDaveHello** [comment]
Hey @Piicksarn let's focus on the menu issue here. You are talking about another issue,

**Comment by Klo1029** [comment]
Used new font cause old font didn't looked good as logo.
And as I know jquery 1.7.2 version doesn't support new features and use old syntax.
I think I've not seen any dropdown navigation property in bootstrap, Also tried many variation to make navigation responsive. Well I don't wanted to mess with the real design of website that's why I used as much css rules as I have to in order to keep site real design.
"I can design navigation better way for that I've to change layout.html and write html again cause layout.html is not well structured or structured in a way easy to develop further".

**Comment by Klo1029** [comment]
What do you think about that??

**Comment by PeterDaveHello** [comment]
@Klo1029 that's different issue, we should better separate them into different issues, mixing them in unrelated issue is not a good idea.

**Comment by PeterDaveHello** [comment]
any updates?

**Comment by Klo1029** [comment]
Am Sorry, I've been busy lately - can you please explain what changes should I make?

**Comment by PeterDaveHello** [comment]
Make the things inconsistent, the changes should be the same as your commit message, and do one thing, one time, so that we can discuss and audit it, thanks.

**Comment by Klo1029** [comment]
ok sure ! I'll keep updates posted.

**Comment by PeterDaveHello** [comment]
@Klo1029 Thanks!

**Comment by PeterDaveHello** [comment]
any updates?

**Comment by Klo1029** [comment]
Well, Am gladly looking forward to it, Sorry I was so busy with some work in past few weeks.

I'll give an update about it very soon...

**Comment by PeterDaveHello** [comment]
Oh no problem, never mind, just make sure this PR is valid.

**Comment by MattIPv4** [comment]
@AmJustSam Hi, sorry this got so abandoned. If you're still active, any chance you could resolve the conflicts so we can merge this? :)

**Pull request closed without merging**